gtwrap/pybind_wrapper.py

```diff
@@ -177,9 +177,9 @@ def _wrap_dunder(self,
         args_signature_with_names = self._method_args_signature(method.args)
 
         if method.name == 'len':
-            function_call = "return self->size();"
+            function_call = "return std::distance(self->begin(), self->end());"
         elif method.name == 'contains':
-            function_call = f"return self->find({method.args.args_list[0].name}) != self->end();"
+            function_call = f"return std::find(self->begin(), self->end(), {method.args.args_list[0].name}) != self->end();"
```

Comment:
Sorry for the back-and-forth, but realizing that std::find(..) is always O(n) whereas for many data structures (e.g. std::set) `self->find(obj)` should be O(1), so probably using the `->find()` version would be better in this case?  Overkill approach I guess would be SFINAE, but that seems excessive in this case.

Reply:
Make up your mind, my guy 😂

I figured the find method would be faster but whoever designed the STL didn't think to make find a method for vectors and lists.

Also, this feature is not crucial. If people want efficiency, then cast to a Python set object (which is easily doable now since we support iteration).